src/components/shared/misc/ImageUploader.vue

```diff
@@ -195,7 +195,7 @@
             },
 
             uploadFile() {
-                if (this.image === null || this.image.indexOf(this.defaultSrc) > -1) {
+                if (this.image === null || this.image.indexOf(this.getDefaultSrc) > -1) {
```

Comment:
This statement returns 0 if `this.getDefaultSrc` is an empty string, is that right?

```js 
this.image.indexOf(this.getDefaultSrc)
```